FirstTask.py

```diff
@@ -0,0 +1,230 @@
+        return (len(self.bank_accounts))
+
+    def create_account(self, name, code, account_type):
+        self.bank_accounts.append(BankAccount(name, code, 1000+len(self.bank_accounts)+1, 0, account_type))
+
+    def find_account(self, account_number):
+        for i in range(len(Bank.bank_accounts)):
+            if account_number == Bank.bank_accounts[i].account_number():
+                return True, Bank.bank_accounts[i]
+        return False, "Account not found!"
+
+    def login(self, account_number, code):
+        status, response = self.find_account(account_number)
+        if status:
+            if code == response.pass_code():
+                self.logged_in_account = response
+                Bank.logged_in = True
+                return True, response
+        return False, "Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        status_1, response_1 = self.find_account(sender_account_number)
+        status_2, response_2 = self.find_account(receiver_account_number)
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True
+        else:
+            return False
+    
+    def compare(self, current_account, other_account):
+        status_1, response_1 = self.find_account(current_account)
+        status_2, response_2 = self.find_account(other_account)
+        if status_1 and status_2:
+            if response_1.check_balance() > response_2.check_balance():
+                return True, 1
+            if response_1.check_balance() < response_2.check_balance():
+                return True, -1
+            if response_1.check_balance() == response_2.check_balance():
+                return True, 0
+        else:
+            return False, f"{other_account} Account Not Found!"
+    
+   
+    def account_details(self, account_number):
+        status,response = self.find_account(account_number)
+        if status:
+            response.account_details()
+        else:
+            return False
+   
+    def logout(self):
+        self.logged_in = None
+        return self.logged_in
+
+    def is_logged_in(self):
+        if self.logged_in:
+            return True
+        return False
+
+def run():
+    account_number_id = 1000 # Wrirting here just to print account number for user after account creation 
+    bank = Bank()
+    while(1):
+        option = int(input("1) Create Account \n2) Login \n3) Find Account \n4) Total Accounts\n0) Exit \nChoose:"))
+        if option == 1:
+            sys('clear')
+            account_number_id += 1
+            name = input("Enter Full Name: ")
+            pass_code = int(input("Enter Passcode "))
+            while(1):
+                ACCOUNT_TYPES = ["Current","Saving","Business"]
+                account_select = int(input("Select Account Type from following:\n1) Current \n2) Saving \n3) Business\n"))

```

Comment:
After moving the `ACCOUNT_TYPES` to the `Bank` class we need to iterate the list of types it can offer here. You can use this code here:
```suggestion
                msg = "\n".join(f"{index + 1}) {account_type}" for index, account_type in enumerate(a))
                account_select = int(input(f"Select Account Type from following:\n{msg}\n"))
```
Also, try to understand what it does.